tidb-lightning/tidb-lightning-compatibility-and-scenarios.md

```diff
@@ -19,6 +19,8 @@ summary: 了解 IMPORT INTO 和 TiDB Lightning 与日志备份和 TiCDC 的兼
 
 - `IMPORT INTO` 与日志备份以及 TiCDC 均不兼容。原因是 `IMPORT INTO` 的导入过程也是将源数据编码后的 KV Pairs 直接 Ingest 到 TiKV。
 
+Lightning 和 TiCDC 共同工作时的操作步骤，参考 [TiCDC 和 Lightning 的兼容性](/ticdc/ticdc-compatibility.md#ticdc-和-lightning-的兼容性)
```

Comment:
```suggestion
- 如果你需要在同一个集群上同时使用 TiDB Lightning 和 TiCDC，请参考 [TiCDC 与 TiDB Lightning 的兼容性](/ticdc/ticdc-compatibility.md#ticdc-与-tidb-lightning-的兼容性)。
```

Please move this note about TiDB Lightning and TiCDC compatibility to line 19, as it should be grouped with the other TiDB Lightning-related content in lines 16-18.